src/components/AssetTree/AssetTree.tsx

```diff
@@ -105,16 +95,28 @@ export class AssetTree extends Component<AssetTreeProps, AssetTreeState> {
   }
 
   async componentDidMount() {
-    const assets = await sdk.Assets.list({ depth: 1 });
+    if (!this.context) {
+      console.error(ERROR_NO_SDK_CLIENT);
+      return;
+    }
+    const assets = await this.context.assets.list().autoPagingToArray();
     this.setState({
-      assets: assets.items,
+      assets,
       treeData:
-        assets && assets.items.length > 0
-          ? AssetTree.mapDataAssets(assets.items)
-          : [],
+        assets && assets.length > 0 ? AssetTree.mapDataAssets(assets) : [],
     });
   }
 
+  cursorApiRequest = async (assetId: number): Promise<Asset[]> => {
+    const result = await this.context!.assets.list({
+      filter: { parentIds: [assetId] },
+    }).autoPagingToArray();
+    if (!result && !Array.isArray(result)) {
+      console.error(ERROR_API_UNEXPECTED_RESULTS);
+    }
+    return [...result];
```

Comment:
Why do we clone the array here?

Reply:
Nope, it's not necessary, changed to return fetched array.